game-core/src/main/java/games/strategy/triplea/attachments/AbstractConditionsAttachment.java

```diff
@@ -58,25 +59,19 @@ protected AbstractConditionsAttachment(final String name, final Attachable attac
   /**
    * Adds to, not sets. Anything that adds to instead of setting needs a clear function as well.
    */
-  @Override
   @GameProperty(xmlProperty = true, gameProperty = true, adds = true)
   public void setConditions(final String conditions) throws GameParseException {
```

Comment:
Looks like this method can now be protected since it is no longer inherited from the interface.